fineract-core/src/main/java/org/apache/fineract/infrastructure/configuration/exception/GlobalConfigurationException.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.infrastructure.configuration.exception;
+
+public class GlobalConfigurationException extends RuntimeException {
+
+    public GlobalConfigurationException(final String name) {
+        super("Global configuration with name:" + name
```

Comment:
we'd better format it like `"with name: '" + "'"` - also we could refer to the regex in the exception message, because that way the validating regex and the textual description can change independently

Reply:
'' and extra space noted, i dont think regexp would be more descriptive than this message.